docs/standard/frameworks.md

```diff
@@ -101,11 +78,49 @@ A framework is typically referenced by a short TFM. A TFM value can represent on
 | Universal Windows Platform | uap          | uap [uap10.0]                                |
 |                            |              | uap10.0 [win10] [netcore50]                  |
 
-## Deprecated frameworks
+## How to specify target frameworks
+
+Target frameworks are specified in your *csproj* project file. When a single target framework is specified, use the **\<TargetFramework>** element. The following console app project file demonstrates how to target the `netcoreapp1.0` TFM:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
 
-The following frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+  </PropertyGroup>
 
-| Deprecated framework | Replacement |
+</Project>
+```
+
+The following library project file targets newer APIs of .NET Standard (`netstandard1.4`) and older APIs of the .NET Framework (`net40` and `net45`). When targeting more than one target framework, use the plural **\<TargetFrameworks>** element. Note how the use of `Condition` attributes includes implementation-specific package references when the library is compiled for the two .NET Framework TFMs:
```

Comment:
> I think we use code styling rather than bold for XML

They list XML schema elements in bold ... but not with &lt; and &gt; in *Formatting developer text elements*. I may have also seen it done with the angle brackets in some MSDN topics and/or was recalling what they want for "markup" elements, which doesn't apply here. imo it renders nice like this, but rules are rules. @mairaw can u confirm if mentioning an XML element in-text is code, bold, or bold with angle brackets?

Ref: *Styleguide/Read?id=2700&topicid=36402*